src/test/java/stepDefinitions/SEDTweaksSteps.java

```diff
@@ -12,7 +26,171 @@ public SEDTweaksSteps()
 		
 	}
 	
+	@Given("^I am on a listing's details pages$")
+	public void i_am_on_a_listing_s_details_pages() throws Throwable {
+	    driver.get("https://chpl.ahrqstg.org/#/product/8481");
+	    WebDriverWait wait = new WebDriverWait(driver, 20);
+		driver.manage().window().maximize();
+		String actualTitle = driver.getTitle();
+		System.out.println("page title is : "+actualTitle);
+	    
+	}
+
+	@When("^I look at the accordions$")
+	public void i_look_at_the_accordions() throws Throwable {
```

Comment:
This doesn't actually do anything, right? It just "system.out"s a bunch of times? First, I think the "System.out"s should be removed from "complete" code; fine for testing, but shouldn't be there in a PR. Second, I think this step itself has no real purpose. I'd rather have the Scenario written without the step at all. I think every step should either _do_ or _assert_ something.